backend/danswer/db/models.py

```diff
@@ -1108,6 +1109,20 @@ class SlackBotConfig(Base):
     persona: Mapped[Persona | None] = relationship("Persona")
 
 
+class SlackApp(Base):
```

Comment:
Do we want any uniqueness constraints here? Like if the bot and app tokens are the same across two apps, then it should not be allowed?